inference-engine/tests/ie_test_utils/functional_test_utils/include/functional_test_utils/path_utils.hpp

```diff
@@ -0,0 +1,49 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <regex>
+
+#include "common_test_utils/file_utils.hpp"
+
+namespace FuncTestUtils {
+static std::vector<std::string> getXmlPathsFromFolderRecursive(const std::vector<std::string> &folderPaths) {
```

Comment:
I suppose that this a redundant function since you can just add search regex pattern with default `.*` value into the `directoryFileListRecursive`

Reply:
fixed